src/components/ReviewConfirm/Certify.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <v-card flat id="step-5-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="[Default Completing Party value]"
+            hint="Enter the Name of the Completing Party"
+            :value="certifiedBy"
+            @input="emitCertifiedBy"
+          />
+        </div>
+      </div>
+      <v-checkbox
+        :value="isCertified"
+        @change="emitIsCertified"
+      >
+        <template slot="label">
+          <div class="certify-stmt">
+            I, <b>{{trimmedCertifiedBy || '[Legal Name]'}}</b>, {{certifyStatementResource.certifyStatementHeader}}
+          </div>
+        </template>
+      </v-checkbox>
+      <p>
+        <ul class="certify-statements">
+          <li v-for="(statement, index) in certifyStatementResource.certifyStatements" v-bind:key="index">
+              {{ statement }}
+          </li>
+        </ul>
+      </p>
+      <p></p>
+      <p class="certify-date">Date: {{date}}</p>
+      <p class="certify-clause">
+        {{certifyStatementResource.certifyClause}}
+      </p>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+import { CertifyStatementIF } from '@/interfaces'
+
+@Component({})
+export default class Certify extends Vue {
+  // Props passed into this component.
+  @Prop({ default: '' })
+  private date: string
+
+  @Prop({ default: '' })
+  private certifiedBy: string
+
+  private isCertified: boolean = false
+
+  @Prop({ default: () => {
+    return {}
+  }
+  })
+  private certifyStatementResource: CertifyStatementIF
+
+  /**
+   * Lifecycle callback to always give the parent a "valid" event for its property values.
+   */
+  private created (): void {
+    this.emitValid(Boolean(this.trimmedCertifiedBy && this.isCertified))
+  }
+
+  /**
+   * Computed value.
+   * @returns The trimmed "Certified By" string (may be '').
+   */
+  private get trimmedCertifiedBy (): string {
+    // remove repeated inline whitespace, and leading/trailing whitespace
+    return this.certifiedBy && this.certifiedBy.replace(/\s+/g, ' ').trim()
+  }
+
+   @Emit('certifiedByChange')
```

Comment:
According to https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier, you should have been able to retain the previous code `@Emit('update:certifiedBy')` and then changed the parent to `v-on:update:certifiedBy="onCertifiedByChange($event)"` (or `@update:certifiedBy...`). I recommend you try this as it should result in cleaner code (ie, prop name = event name).

Reply:
OK